packages/wdio-crossbrowsertesting-service/tsconfig.prod.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "baseUrl": ".",
     "outDir": "./build",
-    "rootDir": "./src"
+    "rootDir": "./src",
+    "types": ["webdriverio", "@wdio/logger", "jest", "@wdio/mocha-framework", "@wdio/jasmine-framework", "@wdio/cucumber-framework"]
```

Comment:
same
```suggestion
    "types": ["webdriverio"]
```

Reply:
We need atleast `@wdio/mocha-framework` and `@wdio/jasmine-framework` to be included because few of the types in `browser config` interface comes from these in the service.ts file